rust/tools/authorization-logic/BUILD

```diff
@@ -0,0 +1,98 @@
+#-------------------------------------------------------------------------------
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#-------------------------------------------------------------------------------
+package(default_visibility = ["//visibility:public"])
+
+load("@rules_rust//rust:rust.bzl", "rust_binary", "rust_library")
+
+licenses(["notice"])
+
+filegroup(
+    name = "antlr4rust_jar",
+    srcs = ["third_party/antlr4-rust/antlr4rust.jar"]
+)
+
+filegroup(
+    name = "antlr_gen_mod_rs",
+    srcs = ["src/parsing/antlr_gen/mod.rs"]
+)
+
+# Generate the parsing-related files using antlr4rust
+genrule(
+    name = "antlr_gen_srcs",
+    srcs = ["src/parsing/AuthLogic.g4"],
+    tools = [
+        ":antlr4rust_jar",
+	":antlr_gen_mod_rs",
+        "@bazel_tools//tools/jdk:current_java_runtime",
+        "@bazel_tools//tools/jdk:java",
+    ],
+    outs = [
+       "antlr_gen/authlogiclexer.rs",
+       "antlr_gen/authlogiclistener.rs",
+       "antlr_gen/authlogicparser.rs",
+       "antlr_gen/lib.rs",
+    ],
+    cmd = "pwd; $(execpath @bazel_tools//tools/jdk:java) " +
```

Comment:
Why the `pwd`? Is this for letting someone know in the build log what is happening? If so, shouldn't this be part of a larger `echo` command providing some context?

Reply:
This was an oversight. I removed it.

Reply:
This was left over from debugging. Removed it now.